snowpack/src/build/import-resolver.ts

```diff
@@ -119,3 +120,37 @@ export function createImportResolver({fileLoc, config}: {fileLoc: string; config
     return false;
   };
 }
+
+/**
+ * Create a import glob resolver function, which converts any import globs relative to the given file at "fileLoc"
+ * to a local file. These will additionally be transformed by the regular import resolver, so they do not need
+ * to be finalized just yet
+ */
+export function createImportGlobResolver({fileLoc, config}: {fileLoc: string; config: SnowpackConfig}) {
+  return async function importGlobResolver(spec: string): Promise<string[]> {
+    if (spec.startsWith('/')) {
+      spec = path.posix.join(config.root, spec);
+    }
+
+    const aliasEntry = findMatchingAliasEntry(config, spec);
+    if (aliasEntry && (aliasEntry.type === 'path')) {
+      const {from, to} = aliasEntry;
+      spec = spec.replace(from, to);
+      spec = path.posix.resolve(config.root, spec);
+    }
+
+    if (!(spec.startsWith('/') || spec.startsWith('.'))) {
```

Comment:
If this is now a local file URL, Windows will have resolved to `D:\\` or `C:\\` or something. We need to check for that here. This is the spot where Windows is throwing: it’s successfully resolved a local file path, but it doesn’t start with a `/` or a `.` so it throws.

Reply:
I think I got this working in this test commit: https://github.com/snowpackjs/snowpack/pull/2889/commits/c78ecfdb6024f3a1acd6aa53b8ee73def1a8f25f#diff-604860e9d94cc77e54e2aa79fd0102cc989cb812b4f4f623f6ff4cfd10644995R130

Feel free to rewrite this as you see fit. It’s tough to explain step-by-step, but the main gist is that:

1. As soon as we start resolving files, let the OS determine the slash type (i.e. `\` on Windows)
2. Assume that we’re working in the OS’ native slash type, up until the very end, through resolutions, `path.join()`, etc.
3. At the very end, use `slash()` for glob, and use `slash()` before returning the value back to Snowpack. This will convert everything to `/` only.